pages/docs/concepts/asyncapi-document/adding-channels.md

```diff
@@ -0,0 +1,179 @@
+
+## Channel Server Relationship
+
+Channels and [servers](../server.md) have a close relationship in asynchronous messaging systems. Channels act as message communication pathways, while servers generate and publish messages to these channels. Channels serve as logical destinations where messages are organized based on purpose or topic. Acting as producers, Servers generate messages and publish them to specific channels. Consumers, including servers, send to channels to receive messages. Channels ensure that messages are routed to the appropriate consumers based on their subscriptions. This relationship between channels and servers forms the foundation for building scalable and flexible messaging systems.
+
+Here is a flowchart to show the relationship between channels and servers:
+
+```mermaid
+graph TD
+  subgraph Servers
+    s1(Servers)
+    s2(Servers)
+  end
+
+  subgraph Channels
+    c1{Channels}
+  end
+
+  style c1 fill:#47BCEE,stroke:#47BCEE;
+
+  s1 --> c1
+  s2 --> c1
+  c1 --> s1
+  c1 --> s2
+```
+
+In this flowchart, channels serve as communication pathways, receiving messages from servers and routing them to consumers, while servers generate and publish messages to these channels.
+
+### Channel Availability on Specific Servers
+
+A channel can be designed to be available on all servers, which means that the channel is globally accessible, allowing messages published to it to reach all connected clients, regardless of their server connection.
+
+Here is a flowchart showing a channel is available on multiple servers and accessible by multiple clients:
+
+```mermaid
+graph TD
+  subgraph Clients
+    c1(Client 1)
+    c2(Client 2)
+    c3(Client 3)
+  end
+
+  subgraph Servers
+    s1(Server 1)
+    s2(Server 2)
+    s3(Server 3)
+  end
+
+  subgraph Channel
+    ch(Channel)
+  end
+
+  style ch fill:#47BCEE,stroke:#47BCEE;
+
+  c1 --- ch
+  c2 --- ch
+  c3 --- ch
+
+  s1 --- ch
+  s2 --- ch
+  s3 --- ch
+```
+
+In the above flowchart, messages published to the channel to reach all connected clients, regardless of their server connection.
+
+Here is an example of how you might specify that a channel is available only on specific servers:
+
+```yml
+channels:
+  userSignedUp:
+    address: user/signedup
+    messages:
+      userSignedUp:
+        description: An event describing that a user just signed up.
+servers:
+  serverA:
+    url: serverA.example.com
+  serverB:
+    url: serverB.example.com
+```
+
+The above code defines a channel `user/signedup` for subscribing to user signup events and two servers, `Server A` and `Server B`, with their respective URLs, enabling clients to receive user signup notifications from either server.
+
+Specifying a channel and servers allows the AsyncAPI document to clarify where each channel can be found. This method is particularly beneficial in complex systems where different servers might be responsible for different operations.
```

Comment:
Remember we've discussed that we want to improve our writing by trying to not start sentences with "this", because that is an undefined antecedent.  Let's rewrite this and all sentences starting with "this". ✌🏽